ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -359,6 +372,28 @@ public void setGravity(int gravity) {
         }
     }
 
+    /**
+     * Enables/disables the flag to animate measurement changes. If this flag is enabled, any
+     * animation that changes the content's text width (e.g. 9999 -> 10000) will have the view's
+     * measured width animated along with the text width. However, a side effect of this is that
+     * the entering/exiting character might get truncated by the view's view bounds as the width
+     * shrinks or expands.
```

Comment:
Let's add a warning in here that this causes a measure/layout pass for each animation frame and may cause performance to degrade.